+ source ../common/storage_disk_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environment preparation.'
+ message='Start environment preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environment preparation.'
Sun Aug 13 14:39:49 2023 - INFO  - Start environment preparation.
+ check_free_disk
++ TEST_DISK
++ id=1
++ python3 /root/mugen/libs/locallibs/get_test_device.py --device disk --node 1
+ disks=
+ disk_list=($disks)
+ local_disk=
+ local_disk1=
+ local_disk2=
+ local_disk3=
+ pvcreate -y /dev/
  No device found for /dev/.
+ vgcreate openeulertest /dev/
  No device found for /dev/.
+ lvcreate -y -L 100M -T openeulertest/mythinpool
  Volume group "openeulertest" not found
  Cannot process volume group openeulertest
+ lvcreate -y -V 1G -T openeulertest/mythinpool -n thinvolume
  Volume group "openeulertest" not found
  Cannot process volume group openeulertest
+ LOG_INFO 'Environmental preparation is over.'
+ message='Environmental preparation is over.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Environmental preparation is over.'
Sun Aug 13 14:40:03 2023 - INFO  - Environmental preparation is over.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase!'
+ message='Start executing testcase!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase!'
Sun Aug 13 14:40:05 2023 - INFO  - Start executing testcase!
+ lvcreate -y -s --name mysnapshot1 openeulertest/thinvolume
  Volume group "openeulertest" not found
  Cannot process volume group openeulertest
+ CHECK_RESULT 5
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_lvm_create_thinSnapshot.sh line 33'
+ message='oe_test_storage_lvm_create_thinSnapshot.sh line 33'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_lvm_create_thinSnapshot.sh line 33'
Sun Aug 13 14:40:07 2023 - ERROR - oe_test_storage_lvm_create_thinSnapshot.sh line 33
+ '[' 0 -eq 1 ']'
+ return 0
+ lvcreate -y -s --thinpool openeulertest/mythinpool thinvolume --name mythinsnap
  Volume group "openeulertest" not found
  Cannot process volume group openeulertest
+ CHECK_RESULT 5
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_lvm_create_thinSnapshot.sh line 35'
+ message='oe_test_storage_lvm_create_thinSnapshot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_lvm_create_thinSnapshot.sh line 35'
Sun Aug 13 14:40:09 2023 - ERROR - oe_test_storage_lvm_create_thinSnapshot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ lvcreate -y -s openeulertest/mythinsnap --name my2ndthinsnap
  Volume group "openeulertest" not found
  Cannot process volume group openeulertest
+ CHECK_RESULT 5
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_lvm_create_thinSnapshot.sh line 37'
+ message='oe_test_storage_lvm_create_thinSnapshot.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_lvm_create_thinSnapshot.sh line 37'
Sun Aug 13 14:40:11 2023 - ERROR - oe_test_storage_lvm_create_thinSnapshot.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ lvs -o name,lv_ancestors,lv_descendants openeulertest
  Volume group "openeulertest" not found
  Cannot process volume group openeulertest
+ CHECK_RESULT 5
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_lvm_create_thinSnapshot.sh line 39'
+ message='oe_test_storage_lvm_create_thinSnapshot.sh line 39'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_lvm_create_thinSnapshot.sh line 39'
Sun Aug 13 14:40:14 2023 - ERROR - oe_test_storage_lvm_create_thinSnapshot.sh line 39
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End of testcase execution!'
+ message='End of testcase execution!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of testcase execution!'
Sun Aug 13 14:40:15 2023 - INFO  - End of testcase execution!
+ CASE_RESULT 0
+ case_re=0
+ test -z 4
+ test 4 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sun Aug 13 14:40:17 2023 - ERROR - failed to execute the case.
+ exit 4
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Sun Aug 13 14:40:19 2023 - INFO  - start environment cleanup.
+ vgremove -y openeulertest
  Volume group "openeulertest" not found
  Cannot process volume group openeulertest
+ pvremove -y /dev/
  No device found for /dev/.
+ LOG_INFO 'Finish environment cleanup.'
+ message='Finish environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup.'
Sun Aug 13 14:40:22 2023 - INFO  - Finish environment cleanup.
